docs/bootstrap.sh

```diff
@@ -18,7 +18,7 @@ if semver check $REF_NAME; then
   export COMMIT_TAG=$REF_NAME
 fi
 
-function build_and_preview {
+function build_docs {
```

Comment:
separated build from the rest, since we're also building docs on master, the only difference is where they get published and whether they get a GH comment or not